onnxruntime/hosting/protobuf/model_metadata.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
I'm confused... are these from the onnx repo proto3 definitions? Did you just split them into multiple files?

Reply:
Resolved